src/components/comment/__tests__/CommentCard.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { shallow } from "enzyme";
+import CommentCard from "../CommentCard";
+
+// describe("<CommentCard /> rendering", () => {
+//     const wrapper = shallow(<CommentCard />);
+//     it("should be able to render <CommentCard />", () => {
+//         wrapper;
+//     });
+// });
+
+describe("Test CommentCard render", () => {
+    it("CommentCard child component should be able to render", () => {
+        const wrapper = shallow(<CommentCard />);
+        expect(wrapper.find({ children: "<AddComment />" }));
```

Comment:
I am not sure if this line actually tests the rendering of <AddComment />. Normally, when using expect, 
expect(wrapper.find("")).toBe(""),  or expect(wrapper.find("something").toEqual(""), The toBe(), toEqual and other method do the test task.

Reply:
Maybe should use expect(wrapper.find({ children: "<AddComment />" }).exists()).toBe(true);

Reply:
Thanks Lisa， I will try this function: expect(wrapper.find({ children: "" }).exists()).toBe(true);